web-components/pagination/src/pagination.ts

```diff
@@ -0,0 +1,197 @@
+import { LitElement, css, html, unsafeCSS } from 'lit';
+import { customElement, state } from 'lit/decorators.js';
+import { repeat } from 'lit/directives/repeat.js';
+import {
+  ColumnDef,
+  flexRender,
+  getCoreRowModel,
+  getFilteredRowModel,
+  getPaginationRowModel,
+  TableController,
+  PaginationState,
+} from '@tanstack/lit-table';
+import '@carbon/web-components/es/components/data-table/index.js';
+import '@carbon/web-components/es/components/pagination/index.js';
+
+import { makeData } from './makeData.ts';
+import indexStyles from './index.scss?inline';
+
+const styles = css`
+  ${unsafeCSS(indexStyles)}
+`;
+
+type Resource = {
```

Comment:
This type definition already exists in `makeData.ts`, we can consolidate and just use one of them.